pyproject.toml

```diff
@@ -1,8 +1,9 @@
 [tool.poetry]
```

Comment:
This should be [project] not [tool.poetry]

Reply:
I just fixed it.